mix.exs

```diff
@@ -21,6 +21,9 @@ defmodule Perudo.Mixfile do
   # Dependencies listed here are available only for this project
   # and cannot be accessed from applications inside the apps folder
   defp deps do
-    []
+    [
+      {:credo, "~> 0.5", only: [:dev, :test]},
```

Comment:
can we get the blake credo config file from student_events or rejr-api copied over to this app